src/util/print.go

```diff
@@ -25,6 +26,15 @@ func PrintSectionHeader(writer io.Writer, text string) {
 	}
 }
 
+// PrintTimeElapsed prints the time elapsed since startTime
+func PrintTimeElapsed(writer io.Writer, startTime time.Time) {
```

Comment:
Can we make this `PrintCommandFooter` and thoughts on having `PrintCommandHeader` return the start time?

Reply:
Hm, I really don't like conflating printing commands w/time metrics, because now every time we want to print a command header we have to deal with a start time being returned - or we would have to print a command header in order to get a start time. 

If we keep those ideas separate I would almost argue that we should keep it as `PrintTimeElapsed` to be clear printing commands/measuring time aren't tightly coupled.

Reply:
Is this used anywhere aside from in runAndPipe? I agree it is a coupling but I believe these should always be used together so I'd prefer the names to reflect that.

I think we should have this function take in a duration instead of a startTime then (moving the time.Since into runAndPipe

Reply:
It will be used here https://github.com/Originate/exosphere/pull/725, although that PR needs to be updated once this one is shipped.

Okay, I think that makes sense to call if `PrintCommandFooter`. I also moved the `time.Since` into runAndPipe.